app.ts

```diff
@@ -604,6 +612,43 @@ class App {
         return this.gridShowing;
     }
 
+    _showGridLines(gridSize: tilespec.GridSize): void {
+        log("Started drawing grid lines...");
+
+        let gridTiles:Array<StBoxLayout> = [];
+
+        for(let monitorIdx = 0; monitorIdx < activeMonitors().length; monitorIdx++) {
+            const workArea = workAreaRectByMonitorIndex(monitorIdx);
+            if (!workArea) { continue; }
+
+            let tileHeight = workArea.size.height / gridSize.height;
+            let tileWidth = workArea.size.width / gridSize.width;
+            let topOffset = workArea.topLeft().y;
+
+            for(let i = 1; i <= gridSize.width; i++) {
+                for(let u = 1; u <= gridSize.height; u++) {
+                    const newGridWidget = new St.BoxLayout({ style_class: `${theme}__grid_lines_preview` });
+                    gridTiles.push(newGridWidget);
+
+                    Main.uiGroup.add_actor(newGridWidget);
+
+                    newGridWidget.width = tileWidth;
+                    newGridWidget.height = tileHeight;
+                    newGridWidget.x = tileWidth * (i - 1);
+                    newGridWidget.y = topOffset + tileHeight * (u - 1);
+                    newGridWidget.monitor = activeMonitors()[monitorIdx];
```

Comment:
I added this but I have no way to test this out, as I have only one monitor. Maybe you @scherepanov could take a look and guide me to the right direction (or add an edit that would work)?